### PR TITLE
Add some comments about current support for vera

### PR DESCRIPTION
**Pull request by pavoni**
## Proposed change

Vera hubs are no longer actively enhanced by the vendor, and new z-wave devices have not been added for a while.

I'm the sole maintainer of this integration and I've slowly moving to the fantastic z-wave-js - so just wanted to add some words of caution to the documentation.

The platform is very stable - and so current users should be fine unless that want to add new devices.

The HA integration is also very stable - and I'll keep my hard ware to make any minor changes needed to HA.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by pavoni** [comment]
@balloob  just wanted to tag you in case you had a view based on our discord chat.

File: source/_integrations/vera.markdown [hunk 1]
```diff
@@ -27,7 +27,11 @@ ha_platforms:
   - switch
 ---
 
-The [Vera](https://getvera.com/) hub is a controller mainly for connecting to Z-Wave devices.
+The [Vera](https://getvera.com/) hub is a controller mainly for connecting to Z-Wave devices. 
+The supported Vera hubs (Edge, Plus and Secure) are no longer available and the firmware is not being actively enhanced by the vendor. New Z-Wave devices are not directly supported and so are not easy to add to the platform.
+The newer Ezlo hubs use a different firmware and are not supported by this integration.
```
Comment (frenck) [inline source/_integrations/vera.markdown]:
```suggestion

The supported Vera hubs (Edge, Plus and Secure) are no longer available and the firmware is not being actively enhanced by the vendor. New Z-Wave devices are not directly supported and so are not easy to add to the platform.

The newer Ezlo hubs use a different firmware and are not supported by this integration.
```

File: source/_integrations/vera.markdown [hunk 2]
```diff
@@ -27,7 +27,11 @@ ha_platforms:
...
+
+[Z-Wave JS](https://www.home-assistant.io/integrations/zwave_js/) is a better choice for new Z-Wave users or for users wanting support for new Z-Wave devices.
```
Comment (frenck) [inline source/_integrations/vera.markdown]:
```suggestion
[Z-Wave JS](/integrations/zwave_js/) is a better choice for new Z-Wave users or for users wanting support for new Z-Wave devices.
```

**Review by frenck — APPROVED** [review]
Thanks, @pavoni 👍

**Pull request merged**